### ISSUE TITLE
[Bug]: Unable to set runner after importing with missing runner

### ISSUE THREAD

**Issue by NekoiNemo** [bottlesdevs/Bottles#3339]
### Describe the bug

When a bottle gets imported (from a backup or by simple copying it into the bottles dir), if its runner isn't installed - a "Missing Runner" error pops up when opening the bottle. however, for some reason, the Settings/Runner has the most up to date installed runner selected (**but not actually used**), which prevents that runner from being assigned to the bottle properly.

The workaround for it seem to be to select a different Runner (for example `sys-wine`, wait several minutes while prefix is being converted, then open settings again and this time select the correct runner, waiting several more minutes for prefix to be converted once again.

Tl;dr: Runner selector in a bottle with missing Runner doesn't account for that fact and just defaults to the first option in the selector, preventing it from being assigned (when the first option is usually what *needs* to be picked in that situation)

### To Reproduce

1. Import a bottle backup with Runner that isn't installed (let's say, `soda-7.0-8`)
2. Open the bottle (warning will be issued)
3. Go to the Settings and observe that the Runner is set to the newest installed runner (say, `soda-8.0-2`).
4. Try to select that runner and observe how it has no effect (as application thinks it's already selected, for some reason).

### Package

Flatpak from Flathub

### Distribution

Arch Linux

### Debugging Information

```shell
Official Package: true
Version: '51.11'
DE/WM: i3
Display:
    X.org: true
    X.org (port): :0
    Wayland: false
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 6.8.2-arch2-1
Disk:
    Total: 16472170496
    Free: 16472006656
RAM:
    MemTotal: 30.7GiB
    MemAvailable: 22.3GiB
Bottles_envs: null
```

**Comment by stevenre3d** [comment]
Issue still occurs exactly as described using version bottles `51.21`.

I have an older bottle with runner `soda-8.0-2`. When I open its Details I get the popup saying "Missing Runner" and that I need to install it or change it, but when I click Settings the runner is listed as `soda-9.0-1` (the only one I have installed) with no way to change it.

Going into Preferences->Runners and manually installing `soda-8.0-2` did seem to resolve the issue, but simply updating the old bottle to use latest was not an option. Once I had `soda-8.0-2` I was able to change the selected runner in Settings, and it launched as expected.
